sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/Processor/EventProcessorClientLiveTests.cs

```diff
@@ -139,241 +132,144 @@ await using (var consumer = new EventHubConsumerClient(EventHubConsumerClient.De
+                            }
                         }
                     }
-                }
-            }
-        }
-
-        /// <summary>
-        ///   Verifies that the <see cref="EventProcessorClient" /> is able to
-        ///   connect to the Event Hubs service and perform operations.
-        /// </summary>
-        ///
-        [Test]
-        [TestCase(10, 1)]
-        [TestCase(10, 3)]
-        [TestCase(10, 10)]
-        [TestCase(30, 10)]
-        [TestCase(32, 7)]
-        [TestCase(32, 32)]
-        [Ignore("Unstable test. (Tracked by: #7458)")]
-        public async Task PartitionDistributionIsEvenAfterLoadBalancing(int partitions, int eventProcessors)
-        {
-            await using (EventHubScope scope = await EventHubScope.CreateAsync(partitions))
-            {
-                var connectionString = TestEnvironment.BuildConnectionStringForEventHub(scope.EventHubName);
-
-                await using (var connection = new EventHubConnection(connectionString))
-                {
-                    ConcurrentDictionary<string, int> ownedPartitionsCount = new ConcurrentDictionary<string, int>();
-
-                    // Create the event processor manager to manage our event processors.
-
-                    var eventProcessorManager = new EventProcessorManager
-                        (
-                            EventHubConsumerClient.DefaultConsumerGroupName,
-                            connectionString
-                        // TODO: fix test. OwnerIdentifier is not accessible anymore.
-                        // onInitialize: eventArgs =>
-                        //     ownedPartitionsCount.AddOrUpdate(eventArgs.Context.OwnerIdentifier, 1, (ownerId, value) => value + 1),
-                        // onStop: eventArgs =>
-                        //     ownedPartitionsCount.AddOrUpdate(eventArgs.Context.OwnerIdentifier, 0, (ownerId, value) => value - 1)
-                        );
-
-                    eventProcessorManager.AddEventProcessors(eventProcessors);
+                );
 
-                    // Start the event processors.
+            eventProcessorManager2.AddEventProcessors(1);
 
-                    await eventProcessorManager.StartAllAsync();
+            // Start the event processors.
 
-                    // Make sure the event processors have enough time to stabilize.
+            await eventProcessorManager2.StartAllAsync();
 
-                    await eventProcessorManager.WaitStabilization();
+            // Wait for the event processors to receive events.
 
-                    // Take a snapshot of the current partition balancing status.
-
-                    IEnumerable<int> ownedPartitionsCountSnapshot = ownedPartitionsCount.ToArray().Select(kvp => kvp.Value);
-
-                    // Stop the event processors.
+            while (!secondBatchCancellationSource.IsCancellationRequested)
+            {
+                await Task.Delay(500);
```

Comment:
Should this just do `Task.Delay(Timeout.Infinite, secondBatchCancellationSource)` to avoid the spinning?

Reply:
Yes, I like that better!